CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+### Unfixed
+- Remove auto scroll from add story method
+- Add highlight function for when a story be saved
```

Comment:
A single entry, like I mentioned in the previous comment would be enough for this change.

Reply:
Ok, fixed! :nerd_face: